arm/README.md

```diff
@@ -91,6 +91,58 @@ post,
 [Developer's Guide to Auth with Azure Resource Manager API](http://www.dushyantgill.com/blog/2015/05/23/developers-guide-to-auth-with-azure-resource-manager-api/),
 that is also quite helpful.
 
+### Use an authentication file
+
+This SDK also supports authentication with a JSON file containing credentials for the service principal. In the Azure CLI, you can create a service principal and its authentication file with this command:
+
+``` bash
+az ad sp create-for-rbac --sdk-auth > mycredentials.json
+```
+
+THe file lloks like this, in case you want to create it yourself:
+
+``` json
+{
+    "clientId": "ad735158-65ca-11e7-ba4d-ecb1d756380e",
+    "clientSecret": "b70bb224-65ca-11e7-810c-ecb1d756380e",
+    "subscriptionId": "bfc42d3a-65ca-11e7-95cf-ecb1d756380e",
+    "tenantId": "c81da1d8-65ca-11e7-b1d1-ecb1d756380e",
```

Comment:
You should replace all of the values above with comments about what that value is.

`
    "clientSecret":"<whatever>"
`

Reply:
Fixed